content/develop/tutorials/databases/snowflake.md

```diff
@@ -7,151 +7,220 @@ slug: /develop/tutorials/databases/snowflake
 
-SELECT * FROM MYTABLE;
-```
+   <Image alt="AWS screenshot 1" src="/images/databases/snowflake-worksheet-execute.png" />
 
-Before you execute the queries, first determine which Snowflake UI / web interface you're using. The examples below use [Snowsight](https://docs.snowflake.com/en/user-guide/ui-snowsight). You can also use [Classic Console Worksheets](https://docs.snowflake.com/en/user-guide/ui-worksheet) or any other means of running Snowflake SQL statements.
+   <Important>
 
-### Execute queries in a Worksheet
+   If no lines are highlighted and you click the play button, only the line with your cursor will execute.
 
-To execute the queries in a Worksheet, highlight or select all the queries with your mouse, and click the play button in the top right corner.
+   </Important>
 
-<Image alt="AWS screenshot 1" src="/images/databases/snowflake-4.png" />
+1. Optional: To view your new database, above the left navigation, select "**Databases**." Click the down arrows (<i style={{ verticalAlign: "-.25em" }} className={{ class: "material-icons-sharp" }}>expand_more</i>) to expand "PETS" → "PUBLIC" → "Tables" → "MYTABLE."
 
-<Important>
+   <Image alt="AWS screenshot 2" src="/images/databases/snowflake-database-new.png" />
 
-Be sure to highlight or select **all** the queries (lines 1-10) before clicking the play button.
+1. For your use in later steps, note down your role, warehouse, database, and schema. In the preceding screenshot, these are the following:
 
-</Important>
+   ```toml
+   role = "ACCOUNTADMIN"
+   warehouse = "COMPUTE_WH"
+   database = "PETS"
+   schema = "PUBLIC"
+   ```
 
-Once you have executed the queries, you should see a preview of the table in the **Results** panel at the bottom of the page. Additionally, you should see your newly created database and schema by expanding the accordion on the left side of the page. Lastly, the warehouse name is displayed on the button to the left of the **Share** button.
+   Because the SQL statements did not specify a schema, they defaulted to the "PUBLIC" schema within the new "PETS" database. The role and warehouse are trial-account defaults. You can see the role and warehouse used by your worksheet in the upper-right corner, to the left of the "**Share**" and play (<i style={{ verticalAlign: "-.25em" }} className={{ class: "material-icons-sharp" }}>play_arrow</i>) buttons.
 
-<Image alt="AWS screenshot 2" src="/images/databases/snowflake-5.png" />
+   In Snowflake, databases provide storage and warehouses provide compute. When you configure your connection, you aren't explicitly required to declare role, warehouse, database, and schema; if these are not specified, the connection will use your account defaults. You can also change these settings within an active connection if you need to use multiple roles, warehouses, or databases. However, declaring these defaults avoids unintentional selections.
```

Comment:
- Insert a comma after "storage." 
- "If you want to use multiple roles, warehouses, or databases, you can also change these settings within an active connection."